utils/uniset/src/builder.rs

```diff
@@ -447,7 +447,7 @@ impl UnicodeSetBuilder {
     pub fn complement_set(&mut self, set: &UnicodeSet) {
         let inv_list_iter_owned = set
             .as_inversion_list()
-            .as_slice()
+            .as_ule_slice()
```

Comment:
Suggestion: I think you should take the opportunity to clean up some of these call sites.  This one can replace all four of the following lines with just `.iter()`